api/Request.json

```
@@ -5,17 +5,33 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/Request",
         "spec_url": "https://fetch.spec.whatwg.org/#request-class",
         "support": {
-          "chrome": {
-            "version_added": "42"
-          },
-          "chrome_android": {
-            "version_added": "42"
-          },
+          "chrome": [
+            {
+              "version_added": "42"
+            },
+            {
+              "version_added": "40",
+              "version_removed": "42",
+              "partial_implementation": true,
+              "notes": "Only available via Service Workers."
```

```suggestion
              "notes": "Only available in service workers"
```

This is to match the style of "worker_support" descriptions and the capitalization of "service workers" in the prose of https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API

Can you make the same change everywhere?